src/api/route-services/plots-tables.js

```diff
@@ -39,7 +38,7 @@ class PlotsTablesService {
       Key: {
         experimentId: { S: experimentId }, plotUuid: { S: plotUuid },
       },
-      UpdateExpression: 'SET plotData = :plotData, plotType = if_not_exists(plotType, :plotType), config = if_not_exists(config, :config)',
+      UpdateExpression: 'SET plotData = :plotData, config = if_not_exists(config, :config)',
```

Comment:
`PlotType` is set by the front end

Reply:
Do you know why this was there originally?

Reply:
yea, I wrote the first part for the sake of completeness. However, the value of `plotType` is hard-coded in the front end for every plot.